applications/StructuralMechanicsApplication/custom_utilities/constraint_utilities.h

```diff
@@ -0,0 +1,64 @@
+// KRATOS  ___|  |                   |                   |
```

Comment:
please move it to the core, it is certainly general enough

Thanks btw, looks really good